src/main/kotlin/io/github/kryszak/gwatlin/api/achievement/model/Achievement.kt

```diff
@@ -16,6 +16,6 @@ data class Achievement(
         val type: String,
         val flags: List<String>,
```

Comment:
Let's initialize `flags` and `tiers` with empty list too. Might save us revisiting this model later on :smile:

Reply:
Good call, done.